server/src/graphql/mutations/createApp.ts

```diff
@@ -28,17 +31,56 @@ export const createApp: MutationResolvers['createApp'] = async (
   if (isAppNameTaken) {
     throw new Error('App name already taken');
   }
+  // we do git related operations only for apps
+  // that have gitRepoURl
+  let repo;
+  if (input.gitRepoUrl) {
+    const repoData = getRepoData(input.gitRepoUrl);
+    try {
+      const res = await fetch(
```

Comment:
you can use the `octokit/rest` package that will give you correct ts typings :)

Reply:
Resolved.